app/controllers/api/orchestration_templates_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  class OrchestrationTemplatesController < BaseController
+    def delete_resource(type, id, data = {})
+      klass    = collection_class(type)
+      resource = resource_search(id, type, klass)
+      super
+      resource.raw_destroy if resource.kind_of?(OrchestrationTemplateVnfd)
```

Comment:
how long does it normally take to destroy an orchestration template including on like *Vnfd ?